zwave_js_server/model/controller.py

```diff
@@ -295,6 +416,31 @@ async def async_begin_inclusion(
         )
         return cast(bool, data["success"])
 
+    async def async_begin_inclusion_s2(
```

Comment:
Can't we merge this with the existing method and raise a RuntimeError or similar if it's called with the wrong combination of arguments, ie provisioning_info and not S2 strategy?

Reply:
Yeah. I think we should try to stay in line with the zwave-js api as far as possible.

Reply:
shall we remove the `async_begin_inclusion_default` method? For now I've added a deprecation warning. Maybe we leave it there for a bit and remove it in the future?

Reply:
I think we can make a breaking change. We're bumping the minimum server version so that's a breaking change anyway.